benchmarks_gui/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
While Qt5 was supported in 2.8.3 (I think), there was a much simpler interface introduced in 2.8.11 using "imported targets" http://doc.qt.io/qt-5/cmake-manual.html
2.8.12 adds support for -fPIE and -fPIC
I could lower this to 2.8.11 but jade has >=2.8.12, for 2.8.11 see https://github.com/ros-planning/moveit_setup_assistant/pull/109/commits/5172eccc87a53108d09dfc3d17a471e49c449864